examples/simplicial_mesh/elixir_euler_tet_mesh.jl

```diff
@@ -0,0 +1,47 @@
+# !!! warning "Experimental features"
```

Comment:
I think it would be nice if you could rename the new examples folder using the same template we use for the other folders. Thus, you could use two folders such as
- `examples/simplicial_2d_dg`
- `examples/simplicial_3d_dg`

to make it clear that there are both 2D and 3D examples (for DG-type methods).